packages/components/src/post-locked-modal/index.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Modal } from '@wordpress/components';
+import { Component, Fragment } from '@wordpress/element';
+import { withSelect } from '@wordpress/data';
+import { PostPreviewButton, getWPAdminURL } from '@wordpress/editor';
+import './style.scss';
```

Comment:
We moved away from CSS imports inside JS files, we have a root level `style.scss` file where we import all the stylesheets now.

Reply:
ah right, so we can just remove this right?

Reply:
yes